### DISCUSSION TITLE
Drag and Drop API keynotes | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#63] [Announcements]
# Drag and Drop API keynotes | Leo's dev blog – stories, insights, and ideas

Some keynotes to keep in mind when working with Drag and Drop APIs in Javascript

https://www.leohuynh.dev/blog/drag-n-drop-api-keynotes

**Comment by niraj-ghetiya** [comment]
Hi
